examples/array.rs

```diff
@@ -3,23 +3,45 @@
 struct Uart {
     // you can be explicit if you like
```

Comment:
This straggling comment no longer makes sense. Perhaps "These fields are ReadWrite by default" would be better now.